lib/beaker/host_prebuilt_steps.rb

```diff
@@ -523,6 +530,9 @@ def set_env host, opts
         when /freebsd/
           host.exec(Command.new("sudo perl -pi -e 's/^#?PermitUserEnvironment no/PermitUserEnvironment yes/' /etc/ssh/sshd_config"), {:pty => true} )
           host.exec(Command.new("sudo /etc/rc.d/sshd restart"))
+        when /openbsd/
+          host.exec(Command.new("sudo perl -pi -e 's/^#?PermitUserEnvironment no/PermitUserEnvironment yes/' /etc/ssh/sshd_config"), {:pty => true} )
+          host.exec(Command.new("sudo /etc/rc.d/sshd restart"))
```

Comment:
is this any different than the freebsd case above it?  If so, you should be sure to consolidate them like you've been doing with your `(free|open)bsd` matches